spec/indexers/generic_work_indexer_spec.rb

```diff
@@ -94,6 +95,33 @@
         expect(solr_document[width_field]).to eq(width)
         expect(solr_document[height_field]).to eq(height)
       end
+
+      describe "when child work representative is updated" do
+        let(:new_width) { 1000 }
+        let(:new_height) { 2000 }
+        let(:new_file_set) { FactoryGirl.create(:file_set, :public) }
+
+        before do
+          # have to get original work in index, so child work can find it
+          # to update it.
+          work.ordered_members << work.representative
+          work.save!
+        end
+
+        it "updates parent work in index" do
+          child_work.ordered_members << new_file_set
+          IngestFileJob.perform_now(new_file_set, (Rails.root + "spec/fixtures/sample.jpg").to_s, nil)
+          child_work.representative = new_file_set
+
+          indexed_parent = SolrDocument.find(work.id)
+          expect(indexed_parent["representative_original_file_id_tesim"]).not_to include(new_file_set.original_file.id)
```

Comment:
`not_to contain_exactly` does not mean the same thing as `not_to include`. Like it could include those things, but not _exactly_, and still pass. 

I guess I could say I expect it to eq `[]` if that's what I expect. I guess I'll look more into what I expect.